client/redux/user.js

```diff
@@ -13,6 +14,7 @@ export const getUser = (sessionId) => {
   return async (dispatch) => {
     const user = await axios.get(`/api/users/${sessionId}`)
     dispatch(setUser(user.data))
+    if(user.data.cart) dispatch(setCartItems(user.data.cart.cartItems))
```

Comment:
i understand the logic, but i think this needs to be moved to the cart store instead. so in the cart store you would account for action type getUser and if there's cart data then handle it there.

Reply:
Ahhh really good idea. This felt gross, but I couldn't think of the right way.

Reply:
done in commit